scripts/clean-shrinkwrap.js

```diff
@@ -15,29 +15,258 @@
 
 'use strict';
 
+const _ = require('lodash');
+const fs = require('fs');
 const path = require('path');
-const os = require('os');
 const packageJSON = require('../package.json');
```

Comment:
> but its not safe to assume that for fs.writeFile as well)

Ahh, that's the bit I was missing, thanks. So `require` definitely works with both slash-types on Windows?